.changeset/cold-donuts-bow.md

```diff
@@ -0,0 +1,5 @@
+---
+"@appsmithorg/design-system": patch
+---
+
+Enable cross icon
```

Comment:
Can you mention component name? since this goes to release notes, this needs to be a bit more specific